server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/shared/_header.scss

```diff
@@ -13,106 +13,120 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@import "go-variables";
+@import "common";
 
-$menu-hover : #3e3d3d;
-$header-height : 50px;
+$header-height: 50px;
```

Comment:
Or if it's only used in this one file, it can be moved inside the `.app-header` scope, but then even `.logo` would need to be moved inside `.app-header`